Products/TinyMCE/adapters/JSONSearch.py

```diff
@@ -32,21 +32,14 @@ def getSearchResults(self, filter_portal_types, searchtext):
         query = self.listing_base_query.copy()
         query.update({
             'portal_type': filter_portal_types,
-            'sort_on': 'sortable_title',
             'path': '/'.join(self.context.getPhysicalPath()),
-            'SearchableText': searchtext,
+            'Title': searchtext,
         })
         if searchtext:
             plone_layout = self.context.restrictedTraverse('@@plone_layout',
                                                            None)
-            if plone_layout is None:
-                # Plone 3
-                plone_view = self.context.restrictedTraverse('@@plone')
-                getIcon = lambda brain: plone_view.getIcon(brain).html_tag()
-            else:
-                # Plone >= 4
-                getIcon = lambda brain: plone_layout.getIcon(brain)()
```

Comment:
There are still a reference of this method at line 58 (before change) or 51 (after change)